src/components/interaction-designer/block-editors/BlockOutputBranchingExplanations.vue

```diff
@@ -0,0 +1,70 @@
+<!-- v-html is required for displaying markdown -->
+<!-- eslint-disable vue/no-v-html -->
+<template>
+  <dl
+    v-if="definitions"
+    class="block-output-branching-explanations">
+    <template v-for="(definition, i) in definitions">
+      <dt :key="`${i}-dt`">
+        {{ definition.title }}
+      </dt>
+      <dd
+        :key="`${i}-dd`"
+        v-html="definition.description" />
```

Comment:
even we're using `marked`, I think it's still risky to use `v-html`, could we pls sanitize it to avoid XSS attacks (according to their documentation: https://marked.js.org/) ?
see example: https://bitbucket.org/voto/%7Be5920ca7-e0cc-4cc7-af7a-574c04a6e31c%7D/pull-requests/2792#Lresources/js/shared-content/components/ReceiveSharedContentModal.vueT9

cc: @bulatgab

Reply:
Past contextes about v-html issue https://votomobile.slack.com/archives/C8SBQDB29/p1614753731003900, https://viamoinc.atlassian.net/browse/VMO-3599